cmd/influx/config.go

```diff
@@ -117,32 +154,75 @@ func (b *cmdConfigBuilder) cmdCreateRunEFn(*cobra.Command, []string) error {
 }
 
 func (b *cmdConfigBuilder) cmdDelete() *cobra.Command {
-	cmd := b.newCmd("delete", b.cmdDeleteRunEFn, false)
+	cmd := b.newCmd("rm [cfg_name]", b.cmdDeleteRunEFn, false)
+	cmd.Aliases = []string{"delete", "remove"}
+	cmd.Args = cobra.ArbitraryArgs
 	cmd.Short = "Delete config"
+	cmd.Long = `
+	The influx config delete command deletes an InfluxDB connection configuration from
+	the configs file (by default, stored at ~/.influxdbv2/configs).
+
+	Examples:
+		# delete a config
+		influx config rm $CFG_NAME
+
+		# delete multiple configs
+		influx config rm $CFG_NAME_1 $CFG_NAME_2
+
+	For information about the config command, see
+	https://v2.docs.influxdata.com/v2.0/reference/cli/influx/config
+	and
+	https://v2.docs.influxdata.com/v2.0/reference/cli/influx/config/remove`
```

Comment:
@sanderson this is an impacting chagne here for hte docs. I made hte names consistent with the rest of the CLI